docs/howtos/extending/magicgui.md

```diff
@@ -577,6 +588,7 @@ You can also create multiple layers by returning a list of
 {attr}`~napari.types.LayerDataTuple`.
 
 ```python
+from magicgui import magicgui
 from typing import List
```

Comment:
```suggestion
```
Let's remove this, un-needed in python 3.10 and on (napari 0.6.0).
Also need to fix line 583/595 to use `list[napari.types.LayerDataTuple]`